tests/fixtures.py

```diff
@@ -123,13 +123,13 @@ def bypass_api_required(monkeypatch):
 
 
 @pytest.fixture
-def live_database_client(db_cursor) -> DatabaseClient:
+def live_database_client(db_cursor, test_client, session) -> DatabaseClient:
     """
     Returns a database client with a live connection to the database
     :param db_cursor:
     :return:
     """
-    return DatabaseClient(db_cursor)
+    return DatabaseClient(db_cursor, test_client, session)
```

Comment:
At some point further down the line, we may want to consolidate these in some manner so that we're not passing three distinct parameters -- maybe encapsulate them as a single data object, perhaps.

Reply:
Additionally note that, for the integration tests to pass, you will need to update `PsycopgResources.py`, specifically at `setup_database_client`, to account for this.

Reply:
> At some point further down the line, we may want to consolidate these in some manner so that we're not passing three distinct parameters -- maybe encapsulate them as a single data object, perhaps.

I've realized while updating things that passing the `test_client` is unnecessary. The `db_cursor` parameter will likely be removed as well once the code is fully migrated to SQLAlchemy, leaving `session` as the last remaining parameter